### DISCUSSION TITLE
Mobile is broken

### DISCUSSION THREAD

**Discussion by AmySeidman** [community/community#137786] [Mobile]
<details>

I can't access anything 

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.176.0 (140321802)
Device: iPhone SE (2nd generation) (iOS 17.5)
</details>

**Comment by aconsuegra** [comment]
Hi @AmySeidman. thanks for reaching out, and I'm sorry to know about the issues you are experiencing with GitHub Mobile.

Can you offer us more details about what is broken? Can you maybe record a video of what is happening and share here so we can better assess the problem?

Thanks!

**Comment by elisealix22** [comment]
Hi @AmySeidman , I'd love to try and help! Are you seeing any errors? Which screens are you having trouble accessing? It might be worth trying to re-authenticate by logging in again to see if that resolves your issue. Let me know if you are still having trouble after that.

**Comment by AmySeidman** [comment]
thanks I am reinstalling the app, and got busy. I will revert if I still
have issues with a video

On Thu, Sep 5, 2024 at 7:06 AM Elise Alix ***@***.***> wrote:

> Hi @AmySeidman <https://github.com/AmySeidman> , I'd love to try and
> help! Are you seeing any errors? Which screens are you having trouble
> accessing? It might be worth trying to re-authenticate by logging in again
> to see if that resolves your issue. Let me know if you are still having
> trouble after that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/137786#discussioncomment-10558258>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIAZHNUQHVL4MIGVV3ITUCLZVBQOPAVCNFSM6AAAAABNVPRRTKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJVHAZDKOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

_____________________________
"You are today where your thoughts have brought you; you will be tomorrow
where your thoughts take you."
                            - James Allen
Amy Seidman
415-387-0274 ~ land line
415-939-0555 ~ air

PROGRAMMING
www.NobleProfit.com <http://www.nobleprofit.com/>
Connecting Sustainable Business and Finance

www.BFLO.io <http://www.bflo.io/>
Deep Tech for Climate and Sustainability

The only constant is change.

______________________________________
This e-mail is intended only for the use of the individual or entity to
whom or which it is addressed, and may contain information that is
privileged, confidential or exempt from disclosure under applicable federal
or State laws.  If the reader of this message is not the intended
recipient, you are hereby notified that any dissemination, distribution or
copying of this communication is strictly prohibited.  If you have received
this communication in error, please contact us immediately by telephone.
Do not disclose the contents to anyone.
